src/openproduct/producttypen/models/thema.py

```diff
@@ -37,13 +37,15 @@ class Thema(BasePublishableModel):
     class Meta:
         verbose_name = _("thema")
         verbose_name_plural = _("thema's")
+        ordering = ("id",)
 
     def __str__(self):
         return self.naam
 
     def clean(self):
-        disallow_hoofd_thema_self_reference(self, self.hoofd_thema)
+        if self.pk:
```

Comment:
Why is this needed now?

Reply:
When uuid was the pk it was not needed because the pk would be generated before saving, but an Autofield is set after saving the model. (the validation is only needed on an update.)